README.md

```
@@ -1,3 +1,4 @@
+testing
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)